frontend/src/pages/PersonalPage.jsx

```diff
@@ -120,43 +114,44 @@ function PersonalPage(props) {
             success = false;
         });
 
-        // if (success) {
-        //     const userNoteResponseJSON = await userNotesResponse.json();
-        //     setNotes([]);
-        //     if (userNoteResponseJSON.note_reference) {
-        //         for (const noteRef of userNoteResponseJSON.note_reference) {
-        //             setNotes((arr) => [...arr, noteRef]);
-        //         }
-        //         const pdfResponse = await fetch(
-        //             "http://localhost:3000/v1/notes/file/" +
-        //                 notes[0].note_reference,
-        //             {
-        //                 method: "GET",
-        //                 headers: {
-        //                     Authorization:
-        //                         "bearer " +
-        //                         window.localStorage.getItem("authToken"),
-        //                 },
-        //             }
-        //         ).catch((err) => {
-        //             console.error(err);
-        //             success = false;
-        //         });
+        if (success) {
+            const userNoteResponseJSON = await userNotesResponse.json();
+            setNotes([]);
+            if (userNoteResponseJSON.notes) {
+                for (const noteRef of userNoteResponseJSON.notes) {
+                    setNotes((arr) => [...arr, noteRef]);
+                }
+
+                const pdfResponse = await fetch(
+                    "http://localhost:3000/v1/notes/file/" +
+                        notes[0].note_reference,
```

Comment:
Check for the case where the user has no notes
![image](https://user-images.githubusercontent.com/58454072/141786305-1c9d0a29-3c55-41f9-91a3-433018699824.png)

Reply:
it also isn't working when i log into a user that has notes

Reply:
(the get user notes can be removed and done in the next pr)

Reply:
also adding friends doesn't work on the personal page

Reply:
Will done in next pr